app/controllers/concerns/health_careplan.rb

```diff
@@ -32,9 +33,25 @@ def careplan_pdf_coversheet
         template: 'health/careplans/_pdf_coversheet',
         layout: false,
         encoding: 'UTF-8',
-        page_size: 'Letter',
       )
-      CombinePDF.parse(coversheet, allow_optional_content: true)
+      grover_options = {
+        display_url: root_url,
+        displayHeaderFooter: false,
+        timeout: 50_000,
+        format: 'Letter',
+        emulate_media: 'print',
+        style_tag_options: [{ content: inline_stylesheet_link_tag('print'), media: 'print' }],
+        margin: {
+          top: '.5in',
+          bottom: '.5in',
+          left: '.4in',
+          right: '.4in',
+        },
+        wait_until: 'networkidle0',
+        print_background: true,
+      }
+      coversheet = "<html><head><meta charset='UTF-8' /></head><body>#{coversheet}</body></html>"
```

Comment:
```suggestion
      coversheet = wrap_in_html(coversheet)
```

and add
```
private def wrap_in_html(content)
  "<html><head><meta charset='UTF-8' /></head><body>#{content}</body></html>"
end
```

So we can have a single truth for this?